.travis.yml

```diff
@@ -1,31 +1,37 @@
 sudo: false
+
 language: node_js
 
 node_js:
-  - "6.10.2"
-  - "stable"
+  - 6.10.2
+  - node
 
 cache: yarn
 
 before_install:
-  - if [[ `npm -v` != 4* ]]; then npm i -g npm@4; fi
+  - if [[ `npm -v` != 4* ]]; then npm install -g npm@4; fi
```

Comment:
I didn't write this. Jana was the one who updated to `yarn`, but only some of the stuff in the `.travis.yml` and `build.sh` got updated. I don't know the reason behind this, but I am just maintaining the status-quo. If someone else wants to look into whatever the "right thing" is, that'd be great, but it wouldn't be part of this change.